patches/server/0866-Ignore-impossible-spawn-tick.patch

```diff
@@ -0,0 +1,23 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: dannyball710 <dannyball710@gmail.com>
+Date: Sat, 12 Feb 2022 23:42:48 +0800
+Subject: [PATCH] Ignore impossible spawn tick
+
+
+diff --git a/src/main/java/net/minecraft/world/level/BaseSpawner.java b/src/main/java/net/minecraft/world/level/BaseSpawner.java
+index 03726227fdd60e9cf77213d50184abff438e01ef..295fd153bc6d14a6dfecceaaf6c04af54b3fe6aa 100644
+--- a/src/main/java/net/minecraft/world/level/BaseSpawner.java
++++ b/src/main/java/net/minecraft/world/level/BaseSpawner.java
+@@ -76,6 +76,12 @@ public abstract class BaseSpawner {
+     }
+ 
+     public void serverTick(ServerLevel world, BlockPos pos) {
++        // Paper start - Ignore impossible spawn tick
++        if (spawnCount <= 0 || maxNearbyEntities <= 0) {
```

Comment:
Inline this in one line when it's just a return with low complexity to avoid large diff